app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ApplicationRecord
+  belongs_to :user
+
+  validates :title, presence: true, length: { maximum: 25 }
+  validates %i[acousticness danceability energy instrumentalness liveness speechiness tempo valence popularity], presence: true
```

Comment:
I think we cannot put the keys in an array for the validation.
We would need to do something like:
:acousticness, :danceability, :energy, :instrumentalness, :liveness, :speechiness, :tempo, :valence, :popularity, presence: true
![image](https://user-images.githubusercontent.com/81938708/226339958-5f55fcfe-aade-4c75-a7ee-61de75b24eee.png)
![image](https://user-images.githubusercontent.com/81938708/226340066-96ce8768-eca9-4c9d-b4e1-b4a09469b050.png)

Reply:
Another way to validate presence true is with validates_presence_of

validates_presence_of %i[acousticness danceability energy instrumentalness liveness speechiness tempo valence popularity]

Reply:
yep you're right! sorry meant to save this PR as a draft. need to test out the validations then i'll add some screen shots of rails console

Reply:
Everything else seems good. Learned a lot from the enum for the time in Event.